README.md

```diff
@@ -125,6 +125,12 @@ The `options` table accepts the following fields:
 - `locale`: (default: `en_US`)
 - `calendar_type`: (default: `calendar_types.GREGORIAN`)
 
+### now
+
+**syntax:** `now = icu_date.now()`
+
+Get the current date and time in milliseconds from the epoch.
+
```

Comment:
could you please make attention that returned value is double

Reply:
Done